app/src/main/java/com/codepath/travel/activities/HomeActivity.java

```diff
@@ -286,9 +274,6 @@ public void onConfigurationChanged(Configuration newConfig) {
     // Assign action to perform for different navigation drawer item
     public void selectDrawerItem(MenuItem menuItem) {
         switch(menuItem.getItemId()) {
-            case R.id.nav_search:
-                launchSearchUserFragment();
-                    break;
```

Comment:
Just in case, I think we want to keep this and `launchSearchUserFragment` below from Aditi's work.